commit 8bccc7a0340404cc14f831dd9c06d3528b5ebcff
Author: popcornmix <popcornmix@gmail.com>
Date:   Sat Oct 6 16:18:27 2012 +0100

    Remove BUG() call from vchiq that is trigger when alsa is killed

Index: linux-3.2.46/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c
===================================================================
--- linux-3.2.46.orig/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c	2013-07-26 19:36:51.000000000 +0000
+++ linux-3.2.46/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c	2013-07-26 19:37:01.000000000 +0000
@@ -3180,7 +3180,7 @@
 		mode, VCHIQ_BULK_TRANSMIT);
 
 	/* This call is for kernel thread use and should not be interrupted */
-	BUG_ON(status == VCHIQ_RETRY);
+	// dc4: remove as it does happen: BUG_ON(status == VCHIQ_RETRY);
 	return status;
 }
 
